### DISCUSSION TITLE
Front-end technical task

### DISCUSSION THREAD

**Discussion by luizstacio** [FuelLabs/fuels-wallet#129] [General]
### Description

The purpose of this is to evaluate how you create a project from scratch by thinking about the UI, code infrastructure, and use of our current tech stack.

### User story

As a user of the Fuel Network, I would like to have a simple wallet, to manage my ETH Balance and send and receive this ETH.

### Features
- Create a screen to show ETH Balance;
- Enable the user to see the wallet address and copy it;
- Create a feature to Transfer ETH to another Fuel address;
  - User should be able to type a `fuel1...` address on `TextInput`;
  - User can click on a button to transfer;

### Stack
- Use `fuels-ts` SDK; https://fuellabs.github.io/fuels-ts/
- Use `Fuel-UI` to help with UI components (https://github.com/FuelLabs/fuel-ui);
- Use [xstate](https://xstate.js.org/) to manage the data flow;
- This wallet doesn't need to be safe (you can store secrets using localStorage or indexedDB).

### Notes
- You can use our TestNet as the provider: https://node-beta-1.fuel.network/graphql.
- To get some ETH. Use the TestNet faucet: https://faucet-beta-1.fuel.network/.

**Comment by Angelmykl** [comment]
amaizing UI/UX and very fast
